packages/x-date-pickers/src/MonthCalendar/MonthCalendar.tsx

```diff
@@ -274,6 +278,7 @@ export const MonthCalendar = React.forwardRef(function MonthCalendar<TDate>(
             onFocus={handleMonthFocus}
             onBlur={handleMonthBlur}
             aria-current={todayMonth === monthNumber ? 'date' : undefined}
+            aria-label={monthLabel}
```

Comment:
Without this, the label of each of the months was a bit cumbersome in the context of a screen reader. Hearing `Sep` or `Jan` might not be enough to understand what is happening. 🤔